### ISSUE TITLE
some problems about model training

### ISSUE THREAD

**Issue by newExplore-hash** [AliaksandrSiarohin/motion-cosegmentation#37]
hi AliaksandrSiarohin, 
    so awesome project, about model training, i find there is no discriminator (include updating discriminator parameters) in the script of train.py, just have generator and segmentation,
    is there any problem  when i retrain motion-cosegmentation?

**Comment by AliaksandrSiarohin** [comment]
Discriminator is not used for motion segmentation. Moreover it also not strictly needed for Fomm.

**Comment by newExplore-hash** [comment]
> Discriminator is not used for motion segmentation. Moreover it also not strictly needed for Fomm.

yeah i understand this is a reconstruction process, and there is no adversarial loss, in other words, no adversarial learning in this process. so discriminator is not strictly needed for motion segmentation. Is that so? 
And if i retrain this model and add discriminator with adv learning for training process, will it improve the effect?

**Comment by AliaksandrSiarohin** [comment]
I Don't think it will change something.
